<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SudokuClass</class>
 <widget class="QMainWindow" name="SudokuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>353</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Load Grid</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Game">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <widget class="QMenu" name="menuLoad_recent">
     <property name="title">
      <string>Load recent</string>
     </property>
     <addaction name="actionNo_recent_games"/>
    </widget>
    <addaction name="actionNew_random"/>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="menuLoad_recent"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_Hints"/>
    <addaction name="separator"/>
    <addaction name="actionSolve_row"/>
    <addaction name="actionSolve_column"/>
    <addaction name="actionSolve_3x3"/>
    <addaction name="separator"/>
    <addaction name="actionSolve_entire_puzzle"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_Game"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/openFile.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/saveAs.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/exit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Show_errors">
   <property name="text">
    <string>&amp;Show errors</string>
   </property>
  </action>
  <action name="actionSolve_row">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/RowSelect.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Solve &amp;Row</string>
   </property>
  </action>
  <action name="actionSolve_column">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/ColumnSelect.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Solve &amp;Column</string>
   </property>
  </action>
  <action name="actionSolve_3x3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/3x3Select.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Solve 3x3 &amp;grid</string>
   </property>
  </action>
  <action name="actionSolve_entire_puzzle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/GridSelectAll.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Solve Entire &amp;Puzzle</string>
   </property>
  </action>
  <action name="actionShow_Hints">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/GridCellsSelected.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show &amp;Errors</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About this application...</string>
   </property>
  </action>
  <action name="actionNew_random">
   <property name="icon">
    <iconset>
     <normalon>:/Sudoku/Resources/newFile.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionNo_recent_games">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No recent games</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Sudoku.qrc"/>
 </resources>
 <connections/>
</ui>
